ephios/core/templates/core/userprofile_form.html

```diff
@@ -19,6 +19,13 @@ <h1>{% trans "Edit user" %}</h1>
             <h1>{% trans "Add new user" %}</h1>
         {% endif %}
     </div>
+    {% if userprofile %}
+        <div class="float-right d-flex">
```

Comment:
I don't like that the button is so close to the field below and it appears kinda lost up there. Maybe put it next to delete button which triggers a "similar" action? Also I'd prefer to have another color for the button as this now indicates the same destructivity as deleting the user.

Reply:
![grafik](https://user-images.githubusercontent.com/49537067/110935682-0c1f9f80-8330-11eb-9cf7-bb43353a4bd4.png)
I feel like this looks also weird... more ideas? I guess it would be okay though